common/supplemental/supplementalMetadata.xml

```diff
@@ -304,6 +304,7 @@ For terms of use, see http://www.unicode.org/copyright.html
 			<languageAlias type="zsm" replacement="ms" reason="macrolanguage"/> <!-- Standard Malay ⇒ Malay -->
 			<languageAlias type="zyb" replacement="za" reason="macrolanguage"/> <!-- Yongbei Zhuang ⇒ Zhuang -->
 			<languageAlias type="him" replacement="srx" reason="macrolanguage"/> <!-- Himachali ⇒ Sirmauri (= Pahari, Himachali) -->
+			<languageAlias type="mnk" replacement="man" reason="macrolanguage"/> <!-- Mandinka ⇒ Mandingo -->
```

Comment:
@conradarcturus Looking at the history of this file, I see this restores a line you deleted (in https://github.com/unicode-org/cldr/pull/4572), so I've probably made a mistake.

I was trying to follow "Never remove an ID" in https://cldr.unicode.org/development/updating-codes/updating-subdivision-codes